sailthru_content/services/sailthru_translation_service.py

```diff
@@ -97,6 +97,16 @@ def _get_tags_from_property(self, source_array, key, tag_prefix):
 
         return tags
 
+    def _return_sku(self, seats):
+        """ Return dict of sku for a course run. """
+        sku = {}
+        for seat in seats:
+            seat_type = seat.get('type')
+            if seat_type in ['verified', 'professional']:
```

Comment:
Can you set the array `['verified', 'professional']` as a constant somewhere in this file?